package.json

```diff
@@ -36,7 +36,7 @@
   "homepage": "https://github.com/PinkAirship/useDataFetch#readme",
   "peerDependencies": {
     "axios": ">=0.21.1",
-    "react": ">=16.8.0 <18.0.0"
+    "react": ">=16.8.0 <18.2.0"
```

Comment:
can we make this less perscriptive? Like, could we do <18? I don't really know.

Reply:
Updated to check against React 18.

Reply:
Does this mean it needs between 16.8 and 18? Can we just make it anything greater than 16.8? I don't really understand peer dependencies and I want to make sure it's more open and not less.

Reply:
This upward requirement has been removed now.